aptos-move/e2e-move-tests/src/tests/code_publishing.data/pack_stdlib/sources/configs/features.move

```diff
@@ -69,6 +69,16 @@ module std::features {
         is_enabled(APTOS_STD_CHAIN_ID_NATIVES)
     }
 
+    /// Whether the new BLAKE2B-256 hash function natives is enabled.
+    /// This is needed because of the introduction of new native functions.
+    /// Lifetime: transient
+    const BLAKE2B_256_NATIVE: u64 = 5;
```

Comment:
Just a heads up that this might need to be further bumped up given recent changes. We'll have to see.

Reply:
I see no issues here.